src/main/java/songbox/house/service/GoogleAuthenticationService.java

```diff
@@ -8,7 +8,7 @@
 
     Drive getDrive();
 
-    String getRequestAccessUrl(String requestUrl);
+    String getRequestAccessUrl();
```

Comment:
REST project uses this method, I don't think these changes are really need for core. If you need for TG bot pass here some constant - do it in the telegram bot project.

OR, at least, fix REST project, taking into account these changes